easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Java'
+version = '1.8.0_141'
```

Comment:
ugh..  it just feels so wrong to have to go to Oracle for Java..... but I'll do it.

Java is a mess given EasyBuild's need for reproducibility.   Calling it just 1.8 that gets updated would be nice.   Or using Lmod's ranges (x>=1.8 && x<1.9) for modules so old versions could be purged.

I'll submit a new PR after I dirty myself again at Oracle.